toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -1342,6 +1363,41 @@ func provisionUserSSHCerts(installChroot *safechroot.Chroot, user configuration.
 		if err != nil {
 			return
 		}
+
+		logger.Log.Infof("Adding ssh key (%s) to user (%s) .ssh/authorized_users", filepath.Base(pubKey), user.Name)
+		pubKeyData, err = file.ReadLines(pubKey)
+		if err != nil {
+			logger.Log.Warnf("Failed to read from SSHPubKey : %v", err)
+			return
+		}
+
+		// Append to the tmp/authorized_users file
+		for i, s := range pubKeyData {
+			_ = i
```

Comment:
You can just write `for _, s := range pubKeyData` instead, if you're not using the index.

Reply:
Done